cli/test/test-android-onboarding-progress.mjs

```
@@ -0,0 +1,96 @@
+    await fn()
+    console.log(`✅ PASSED: ${name}`)
+    testsPassed++
+  }
+  catch (error) {
+    console.error(`❌ FAILED: ${name}`)
+    console.error(`   Error: ${error.message}`)
+    testsFailed++
+  }
+}
+
+function assertEquals(a, b, msg) { if (a !== b) throw new Error(msg || `Expected ${b}, got ${a}`) }
```

ESLint

[error] This line has 2 statements. Maximum allowed is 1.

(style/max-statements-per-line)